x-pack/qa/build.gradle

```diff
@@ -24,4 +24,18 @@ subprojects {
       include 'rest-api-spec/api/**'
     }
   }
+
+  // We define a specific reposiroty for opensaml since the shibboleth project doesn't publish to maven central and the
+  // artifacts that are located there are not curated/updated by the project
+  // see: https://wiki.shibboleth.net/confluence/display/DEV/Use+of+Maven+Central
+  repositories {
```

Comment:
Perhaps this should be defined in x-pack/build.gradle so it doesn't need to be redefined here and in plugin?

Reply:
I did that for the benefit of exposing the repository in as fewer projects as possible. But in retrospect, given that we explicitly whitelist the groups to include from the repo, the duplication might be worse. I added it to x-pack/build.gradle